packages/preload/src/modules/workspace.ts

```diff
@@ -141,3 +141,30 @@ export async function createProject(name: string): Promise<Project> {
     return project;
   }
 }
+
+/**
+ * Deletes a project directory and all its contents.
+ *
+ * @param _path - The path of the directory to be deleted.
+ * @returns A Promise that resolves when the directory has been deleted.
+ */
+export async function deleteProject(_path: string): Promise<void> {
+  if (import.meta.env.DEV) return;
```

Comment:
is this intended? Having a different state in the editor than in the file system is causing issues for me, becuase after i delete a scene if I try to create a new one it clashes.

Reply:
forgot about this line :P